src/pages/landingPage.js

```diff
@@ -67,31 +98,7 @@ function LandingPage() {
               </UnorderedList>
             </div>
           </Details>
-          <Paragraph>
-            This service can only be used by **council tenants** to request a repair.
-          </Paragraph>
-          <Paragraph linkRenderer={newTabLinkRenderer}>
-            To request a repair for a **communal area**, please see customer
-            services [contact details page](https://www.lincoln.gov.uk/contact-1/get-touch).
-          </Paragraph>
-          <Details summary="Which areas are communal?" className="govuk-!-margin-top-6" data-testid="landing-page-communal-prompt">
-            <div data-testid='landing-page-communal-info'>
-              <Paragraph>
-                Communal repairs are usually in areas that people share.
-              </Paragraph>
-              <Paragraph>
-                They can include:
-              </Paragraph>
-              <UnorderedList>
-                <ListItem>repairs to door entry systems</ListItem>
-                <ListItem>lock repairs to communal doors</ListItem>
-                <ListItem>lighting repairs to shared areas</ListItem>
-                <ListItem>glazing repairs to shared doors or stairway windows</ListItem>
-                <ListItem>roof and gutter repairs</ListItem>
-                <ListItem>structure and external walls to your block</ListItem>
-              </UnorderedList>
-            </div>
-          </Details>
+          <hr></hr>
```

Comment:
Opening hours in the prototype are different to the MVP:
Differences:

- Prototype has `Wednesday 10am - 4.30pm` and MVP has `Wednesday 9am - 4.30pm` 
- Prototype has `Thursday 9am - 5pm` and MVP has `Thursday 10am - 5pm`